src/component-library/LoadingSpinner/LoadingSpinner.style.tsx

```diff
@@ -0,0 +1,49 @@
+import styled from 'styled-components';
+
+import { theme } from 'component-library';
+
+interface BaseLoadingSpinnerProps {
+  diameter?: number;
+  thickness?: number;
+}
+
+const BaseLoadingSpinner = styled.span<BaseLoadingSpinnerProps>`
+  width: ${(props) => props.diameter}px;
```

Comment:
Should we use rem instead of px's?

Reply:
Don't think so in this case :) the spinner isn't relative to anything (i.e. if we bumped up the base font size we wouldn't want the spinner to change) so I absolute units are appropriate